.circleci/config.yml

```diff
@@ -26,20 +26,20 @@ jobs:
             - ./packages/shared/node_modules
             - ./packages/webapp/node_modules
             - ./packages/extension/node_modules
-  build:
-    docker:
-      - image: cimg/node:22.11
-    resource_class: medium+
-    steps:
-      - checkout
-      - restore_cache:
-          keys:
-            - deps-v7-{{ checksum "pnpm-lock.yaml" }}
-            - deps-v7-{{ .Branch }}
-      - run:
-          name: Build
-          command: npm run build:chrome
-          working_directory: packages/extension
+#  build:
+#    docker:
+#      - image: cimg/node:22.11
+#    resource_class: medium+
+#    steps:
+#      - checkout
+#      - restore_cache:
+#          keys:
+#            - deps-v7-{{ checksum "pnpm-lock.yaml" }}
+#            - deps-v7-{{ .Branch }}
+#      - run:
+#          name: Build
+#          command: npm run build:chrome
+#          working_directory: packages/extension
```

Comment:
just burn it all the way with comments?

Reply:
Ah yeah mostly in case we wanted to bring it back?
Happy to fully remove if you prefer

Reply:
Yeah, lets just remove it